examples/azure-ark/porter.yaml

```diff
@@ -5,8 +5,7 @@ mixins:
 name: porter-azure-ark
 description: "A Porter example using Azure Storage and Ark"
 version: 0.1.0
-invocationImage: deislabs/porter-azure-ark:latest
-tag: deislabs/porter-azure-ark-bundle:v0.1.0
+tag: getporter/porter-azure-ark:v0.1.0
```

Comment:
Before we merge this, let's figure out which examples we want to keep so that we don't end up pushing an image that we don't want anymore.

Reply:
(this azure-ark bundle has been deleted)